spid_cie_oidc/provider/tests/test_03_refresh_token.py

```diff
@@ -67,61 +67,61 @@ def setUp(self):
         session = OidcSession.objects.create(
-            user=User.objects.create(username = "username"),
+            user=User.objects.create(username="username"),
             user_uid="",
             nonce="",
-            authz_request={"scope": "offline_access", "prompt": "consent", "nonce": "123", "acr_values":["https://www.spid.gov.it/SpidL2"]},
-            client_id="",
+            authz_request={"scope": "offline_access", "prompt": "consent", "nonce": "123",
+                           "acr_values": ["https://www.spid.gov.it/SpidL2", "https://www.spid.gov.it/SpidL1"]},
+            client_id=RP_SUB,
             auth_code="code",
         )
         IssuedToken.objects.create(
-            refresh_token = self.rt_jws,
-            session = session,
-            expires = timezone.localtime()
+            refresh_token=self.rt_jws,
+            session=session,
+            expires=timezone.localtime()
         )
 
     def test_grant_refresh_token(self):
         client = Client()
         url = reverse("oidc_provider_token_endpoint")
         request = dict(
-            client_id = RP_CLIENT_ID,
-            client_assertion = self.ca_jws,
-            client_assertion_type = "urn:ietf:params:oauth:client-assertion-type:jwt-bearer",
-            refresh_token = self.rt_jws,
-            grant_type="refresh_token",
-            code = "code",
-            code_verifier = "code_verifier"
+            client_id=RP_CLIENT_ID,
+            client_assertion=self.ca_jws,
+            client_assertion_type="urn:ietf:params:oauth:client-assertion-type:jwt-bearer",
+            refresh_token=self.rt_jws,
+            grant_type="refresh_token"
+            # code = "code",
+            # code_verifier = "code_verifier"
 
         )
         res = client.post(url, request)
         self.assertTrue(res.status_code == 200)
         refresh_token = verify_jws(res.json().get("refresh_token"), op_conf_priv_jwk)
-        self.assertEqual(refresh_token["sub"], RP_SUB)
+        self.assertEqual(refresh_token["aud"], RP_SUB)
 
-    @override_settings(OIDCFED_PROVIDER_MAX_REFRESH = 1)
+    @override_settings(OIDCFED_PROVIDER_MAX_REFRESH=1)
     def test_grant_refresh_token_two_times(self):
         client = Client()
         url = reverse("oidc_provider_token_endpoint")
         request = dict(
-            client_id = RP_CLIENT_ID,
-            client_assertion = self.ca_jws,
-            client_assertion_type = "urn:ietf:params:oauth:client-assertion-type:jwt-bearer",
-            refresh_token = self.rt_jws,
+            client_id=RP_CLIENT_ID,
+            client_assertion=self.ca_jws,
+            client_assertion_type="urn:ietf:params:oauth:client-assertion-type:jwt-bearer",
+            refresh_token=self.rt_jws,
             grant_type="refresh_token",
-            code = "code",
-            code_verifier = "code_verifier"
+            code="code",
+            code_verifier="code_verifier"
```

Comment:
we have removed this from the view but we keep these in the test

if removed in the refresh floe they should be removed from the test as well